### ISSUE TITLE
File name convention and entity naming.

### ISSUE THREAD

**Issue by ds10** [jiscdev/analytics-udd#236]
There is some confusion with HEIS with historic naming of entities not matching file names. Rob has some feedback from HEI's. Opening a ticket to raise at this at the monday call.

**Comment by alanepaull** [comment]
This was raised and discussed in #153, which was closed in February. The issue with making a change of this nature is that it would radically affect pretty much all previous work. It was, at that point, not considered a good idea.

**Comment by alanepaull** [comment]
Monday 13 Nov 17: Clarification - this issue is really about API endpoints and entity names, where filenames are a reflection of the latter. Our preference is that documentation matches the API endpoints. However, it will be painful to change the filenames and entity names in any case. Alan to investigate the impact of this change prior to any decisions about action.

**Comment by alanepaull** [comment]
Monday 22 January 2018: Impact assessment of changing the naming showed that this would be both time-consuming and arduous. Instead, it was decided to tackle the issue through revising the documentation. The Core Sections in ReadMe can be revised to make the mapping of the names readily available and easy to see. In addition, @alanepaull to review other documentation to see if it could be improved.

**Comment by alanepaull** [comment]
I've replaced the list of entities with a table of entity name and API endpoint names to the ReadMe. I think this is much clearer now.

In addition, I could add the API endpoint name to each of the code files. See [assessment instance](https://github.com/jiscdev/analytics-udd/blob/v1.3.3/udd/assessment_instance.md) below the list of properties. Let me know if there's a better location for this in the files; I tried a few and this location and format seems best.
